### PR TITLE
Adds icons for Facebook, Facebook Messenger, Twitter

### PR DESCRIPTION
**Pull request by tildedave**
Also updates use of render: function() -> render()

### CODE REVIEW

**Comment by tildedave** [comment]
ping

File: app/stylesheets/FacebookMessengerLogo.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@import 'support/colors';
+
+.tilt-icon-facebook-messenger-logo {
+    width: 20px;
+    height: 18px;
+
+    .facebook-messenger-logo-cloud {
```
Comment (tildedave) [inline app/stylesheets/FacebookMessengerLogo.scss]: maybe `bubble` is more appropriate

File: app/components/FacebookLogo.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const React = require('react');
+const classnames = require('classnames');
+
+module.exports = React.createClass({
+    displayName: 'FacebookLogo',
+
+    render: function() {
+        const classes = classnames('tilt-icon', 'tilt-icon-facebook-logo', {
+            [this.props.className]: !!this.props.className
+        });
```
Comment (marr) [inline app/components/FacebookLogo.js]:
```
const classes = classnames('tilt-icon', 'tilt-icon-facebook-logo', this.props.className);
```

Comment (tildedave) [inline app/components/FacebookLogo.js]: oneat

File: app/components/FacebookMessengerLogo.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    displayName: 'FacebookMessengerLogo',
+
+    render() {
+        const classes = classnames(
+            'tilt-icon',
+            'tilt-icon-facebook-messenger-logo',
+            {
+                [this.props.className]: !!this.props.className
+            }
+        );
```
Comment (marr) [inline app/components/FacebookMessengerLogo.js]: `const classes = classnames('tilt-icon', 'tilt-icon-facebook-messenger-logo', this.props.className);`

**Comment by tildedave** [comment]
ping updated

**Comment by marr** [comment]
:+1: after squash

**Comment by tildedave** [comment]
ping now with gmail

**Comment by marr** [comment]
:+1:

**Pull request merged**